src/js/health-records/reducers/refresh.js

```diff
@@ -0,0 +1,22 @@
+import _ from 'lodash';
+
+const initialState = {
+  statuses: {
+    OK: [],
+    ERROR: [],
+  },
+};
+
+export default function refresh(state = initialState, action) {
+  switch (action.type) {
+    case 'REFRESH_POLL_SUCCESS': {
+      // returns group in form {'OK': [], 'Error': []}
+      const statuses = _.groupBy(action.data, 'attributes.status');
```

Comment:
Since it seems we're only looking at the lengths of the groups of statuses, would it make more sense to use `_.countBy`? Is the data going to be used for something else, maybe error handling?